google/cloud/security/common/gcp_api/compute.py

```diff
@@ -72,7 +140,28 @@ def get_backend_services(self, project_id):
                     previous_request=list_request,
                     previous_response=response)
         except (HttpError, HttpLib2Error) as e:
-            raise api_errors.ApiExecutionError('backend_services', e)
+            raise api_errors.ApiExecutionError(resource_name, e)
+
+    def get_backend_services(self, project_id):
+        """Get the backend services for a project.
+
+        Args:
+            project_id: The project id.
+
+        Yield:
```

Comment:
nit: should be a ```Return``` instead of yield.  I see this occur at least one more time down below. Can you please double check everywhere?